Experimental/Anticipation Sample/Assets/Materials/Player.mat

```diff
@@ -0,0 +1,80 @@
+%YAML 1.1
```

Comment:
On quick inspection, this material is unused as well, and could be removed.

Reply:
It is used by the Player in-scene-placed object.

Reply:
Apparently I forgot to check in the update to the scene where I actually used that material.